backend/Recodo.BLL/Services/CommentService.cs

```diff
@@ -0,0 +1,55 @@
+using AutoMapper;
+using Microsoft.EntityFrameworkCore;
+using System.Threading.Tasks;
+using Recodo.BLL.Services.Abstract;
+using Recodo.Common.Dtos.Comment;
+using Recodo.DAL.Context;
+using Recodo.DAL.Entities;
+using System.Collections.Generic;
+
+namespace Recodo.BLL.Services
+{
+    public sealed class CommentService : BaseService
+    {
+        public CommentService(RecodoDbContext context, IMapper mapper) : base(context, mapper) { }
+
+        public async Task<CommentDTO> CreateComment(NewCommentDTO newComment)
+        {
+            var commentEntity = _mapper.Map<Comment>(newComment);
+
+            _context.Comments.Add(commentEntity);
+            await _context.SaveChangesAsync();
+
+            var createdComment = await _context.Comments
+                .Include(comment => comment.Author)
+                    .ThenInclude(user => user.AvatarLink)
+                .FirstAsync(comment => comment.Id == commentEntity.Id);
+
+            return _mapper.Map<CommentDTO>(createdComment);
+        }
+
+        public async Task DeleteComment(int commentId)
+        {
+            var commentEntity = await _context.Comments.FirstOrDefaultAsync(p => p.Id == commentId);
```

Comment:
It's better to check owner (user) everywhere, as theoretically any user can change any comment by id via postmen